eosdis-nasa/earthdata-pub-dashboard
# Edpub 1254 multi review status component

# Description

Add multi reviewer component to review steps in the dashboard.

## Spec

See Ticket: [EDPUB-1254](https://bugs.earthdata.nasa.gov/browse/EDPUB-1254)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Start a submission using the GHRC workflow.
4. Create test users that a GHRC daac staff.
5. Using the ```submission/operation/createStepReviewApproval``` endpoint add the user to a review step for the new submission.
6. Progress to the review step
7. Using one of the test users approve submission and confirm the workflow does not progress but the reviewers section updates to reflect the review.
8. Using a user that is not a reviewer confirm you are not able to submit a review.
9. Using another user listed as a review reject the submission and confirm the reviewers section updates and the step is not rolled back.
10. Using the same user update the review to approved and have all listed reviewers approved
11. Confirm the submission progresses.
12. Confirm a review step without reviewers progresses normally and does not have a reviewers section.

---

## Change Log

Add multi reviewer component to review steps in the dashboard.

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
